stuff/cron/plagiarism_detector.py

```diff
@@ -174,15 +179,16 @@ def filter_and_format_result(dbconn: lib.db.Connection, contest_id: int,
                 submission_id_1=guid_and_submission_id_dict[os.path.splitext(
                     os.path.basename(result.test_filename))[0]],
                 submission_id_2=guid_and_submission_id_dict[os.path.splitext(
-                    os.path.basename(result.test_filename))[0]],
+                    os.path.basename(result.reference_filename))[0]],
                 contents=json.dumps({
                     'file1': get_range(result[4].split('\n')),
                     'file2': get_range(result[5].split('\n'))
                 }),
             ))
-    # add to the database.
+        # add to the database.
```

Comment:
can this be restored?
```suggestion
    # add to the database.
```

Reply:
Done. I was testing something, didn't revert it back.